code-challenges.test.js

```diff
@@ -14,46 +16,240 @@
 
+//PSEUDOCODE:
+//Create function called divisible
+//input: objest as an argument
+//output: string stating if the number in the object is divisible by 3 or not
+//use conditionals in function
+//if % 3 of the value is strickly equal to zero, then the number is divisible by 3
+//else, the value is not divisible by 3
+//callback function on the 3 variable and run $yarn jest for test
+
+
+const divisible = (value) => {
+    if(value.number % 3 === 0) {
+        return `${value.number} is divisible by three`
+    }
+    else {
+        return `${value.number} is not divisible by three`
+    }
+   
+}
+console.log(divisible(object1))
+//OUTPUT: 15 is divisible by three
+console.log(divisible(object2))
+//OUTPUT: 0 is divisible by three
+console.log(divisible(object3))
+//OUTPUT: -7 is not divisible by three
+
+// 15 is divisible by three
+
+// at Object.log (code-challenges.test.js:80:9)
+
+// console.log
+// 0 is divisible by three
+
+// at Object.log (code-challenges.test.js:82:9)
+
+// console.log
+// -7 is not divisible by three
+
+// at Object.log (code-challenges.test.js:84:9)
+
+// PASS  ./code-challenges.test.js
+// divisible
+// ✓ takes a object as an argument and decides if the number inside it is evenly divisible by three or not (1 ms)
+
+
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// // // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+// // // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+describe ("cap", () => {
+        it("takes in an array of words and returns an array with all the words capitalized",() => {
+        expect (cap(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect (cap(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+        })
+    })
+
+//GOOD FAIL:
+
+// FAIL  ./code-challenges.test.js
+// capitalized
+//   ✕ takes in an array of words and returns an array with all the words capitalized
+
+// ● capitalized › takes in an array of words and returns an array with all the words capitalized
+
+//   ReferenceError: capitalized is not defined
+
+
 
 // b) Create the function that makes the test pass.
 
+//PSEUSO CODE
+//Create a function called capitalized
+//input: array of strings
+//output: array with all the words capitalized
+//use map to iterate
+//after map method has been implemented, use charAt to find the character at index 0 of each element and use toUppercase on the character at index 0
```

Comment:
Your pseudocode is very good! You are hitting competent comprehension level in your pseudocode by displaying 'detailed pseudo code using vocabulary terms'.